test/unit/app/utils/test_utils_user.py

```diff
@@ -18,7 +19,14 @@
 #     assert create_transfer_account_user.one_time_code is not None
 #     assert create_transfer_account_user.transfer_account is not None
 from helpers.factories import UserFactory, OrganisationFactory, TokenFactory, TransferAccountFactory, fake
-from server.utils.user import transfer_usages_for_user, send_onboarding_sms_messages, admin_reset_user_pin, proccess_create_or_modify_user_request
+from server.models.user import RegistrationMethodEnum
```

Comment:
Oh please on one line, rather.

In general when we need so many symbols from one package that suggests we should have some sort of helper instead perhaps. We can look at that when and if we ever get around to cleaning up.